<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingPage</class>
 <widget class="QDialog" name="settingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="language_label">
     <property name="font">
      <font>
       <family>標楷體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>句子翻譯語言:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="language_selector">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="camera_select_label">
     <property name="font">
      <font>
       <family>標楷體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>鏡頭:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="camera_selector">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="contrast_label">
       <property name="font">
        <font>
         <family>標楷體</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>對比(1.0):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="contrast_scrollbar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="brightness_label">
       <property name="font">
        <font>
         <family>標楷體</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>亮度(0):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brightness_scrollbar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="frameShow_layout">
     <item>
      <widget class="QLabel" name="frameshow_label">
       <property name="font">
        <font>
         <family>標楷體</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>顯示模式:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frameDefault_btn">
       <property name="font">
        <font>
         <family>標楷體</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>影像正常顯示</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frameHorizontal_btn">
       <property name="font">
        <font>
         <family>標楷體</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>影像水平反轉</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frameVertical_btn">
       <property name="font">
        <font>
         <family>標楷體</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>影像垂直反轉</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frameInverse_btn">
       <property name="font">
        <font>
         <family>標楷體</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>影像水平垂直反轉</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="init_btn">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> 設定初始化</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
